### PR TITLE
Update list.json

### PR DESCRIPTION
**Pull request by unifi-protocol**

### CODE REVIEW

File: blockchains/tron/validators/list.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     {
         "id": "TGzz8gjYiYRqpfmDwnLxfgPuLVNmpCswVp",
         "name": "Sesameseed",
-        "description": "Sesameseed is a blockchain community providing fair and transparent representation in delegated governance.",
+        "description": "Sesameseed is a blockchain community providing fair and transparent representation in delegated governance by rewarding Voters for their participation on Tron and Ontology.",
```
Comment (kolya182) [inline blockchains/tron/validators/list.json]:
```suggestion
        "description": "Sesameseed is a blockchain community providing fair and transparent representation in delegated governance by rewarding Voters for their participation.",
```

Comment (kolya182) [reply blockchains/tron/validators/list.json]: Sounds like it rewords on both chains by participating on Tron only

Comment (sesameseed-org) [reply blockchains/tron/validators/list.json]: Ahh...

I understand. Technically it does reward on both chains by participating on Tron Only. The token "SEED" we reward the user in can be redeemed for the representative node rewards on both chains. (e.g. 1 seed = X TRX and Y ONG)

But your change is great, we can use that until other nodes become active

Comment (kolya182) [reply blockchains/tron/validators/list.json]: Once we support ONG staking ,we can came back and change it

**Pull request merged**